### ISSUE TITLE
Consider NoContent action result method on Controller

### ISSUE THREAD

**Issue by PradeepKadubandi** [aspnet/Mvc#3334]
This came up while I and Ryan were briefly discussing code to be generated for Web API scaffolding, it may be handy to define NoContent action result method on Controller class.

**Comment by danilodeven** [comment]
This is already implemented @PradeepKadubandi
```c#
    public IActionResult index() {
        return new NoContentResult();
    }
```
Is this what you are looking for?

**Comment by Eilon** [comment]
I would imagine this is about having a helper factory method on the `Controller` base class, just like there is for many other result types.

**Comment by ryanbrandenburg** [comment]
Fixed in #3547
